tommath.h

```diff
@@ -665,7 +662,12 @@ MP_DEPRECATED(mp_prime_rand) mp_err mp_prime_random_ex(mp_int *a, int t, int siz
 mp_err mp_prime_rand(mp_int *a, int t, int size, int flags) MP_WUR;
 
 /* Integer logarithm to integer base */
-mp_err mp_ilogb(const mp_int *a, mp_digit base, mp_int *c) MP_WUR;
+mp_err mp_ilogb(const mp_int *a, uint32_t base, mp_int *c) MP_WUR;
```

Comment:
this was added since the last release, can't we make it private?

ping @czurnieden

Reply:
Why this is a useful function for the public. No internal helper.